src/components/ProductTours/TourCollapsible.tsx

```diff
@@ -105,16 +75,77 @@ const TourCollapsible: FC<Props> = (
     reopenTour(adminUuid);
   };
 
-  const steps = QUICK_START_GUIDE_STEPS.filter(step => {
-    switch (step.title) {
-      case ADMINISTER_SUBSCRIPTIONS_TITLE:
-        return enableSubscriptionManagementScreen;
-      case CUSTOMIZE_REPORTS_TITLE:
-        return enableReportingConfigScreen;
-      default:
-        return true;
+  useEffect(() => {
+    const QUICK_START_GUIDE_STEPS: StepDefinition[] = [{
+      icon: TrendingUp,
+      title: TRACK_LEARNER_PROGRESS_TITLE,
+      timeEstimate: 2,
+      targetId: TRACK_LEARNER_PROGRESS_TARGETS.LEARNER_PROGRESS_SIDEBAR,
+      completed: false,
+    }, {
+      icon: InsertChartOutlined,
+      title: VIEW_ENROLLMENTS_INSIGHT_TITLE,
+      timeEstimate: 1,
+      targetId: ANALYTICS_INSIGHTS_TARGETS.SIDEBAR,
+      completed: false,
+    }, {
+      icon: CreditCard,
+      title: ADMINISTER_SUBSCRIPTIONS_TITLE,
+      timeEstimate: 2,
+      targetId: ADMINISTER_SUBSCRIPTIONS_TARGETS.SIDEBAR,
+      completed: false,
+    }, {
+      icon: Person,
+      title: ORGANIZE_LEARNERS_TITLE,
+      timeEstimate: 2,
+      targetId: ORGANIZE_LEARNER_TARGETS.PEOPLE_MANAGEMENT_SIDEBAR,
+      completed: false,
+    }, {
+      icon: TextSnippet,
+      title: CUSTOMIZE_REPORTS_TITLE,
+      timeEstimate: 1,
+      targetId: CUSTOMIZE_REPORTS_SIDEBAR,
+      completed: false,
+    }, {
+      icon: Settings,
+      title: SET_UP_PREFERENCES_TITLE,
+      timeEstimate: 1,
+      targetId: TOUR_TARGETS.SETTINGS_SIDEBAR,
+      completed: false,
+    }];
+
+    const FLOW_UUID_MAPPING = new Map([
+      [TRACK_LEARNER_PROGRESS_TITLE, configuration.ADMIN_ONBOARDING_UUIDS.FLOW_TRACK_LEARNER_PROGRESS_UUID?.toString()],
+      [VIEW_ENROLLMENTS_INSIGHT_TITLE, configuration.ADMIN_ONBOARDING_UUIDS.FLOW_ENROLLMENT_INSIGHTS?.toString()],
+      [ADMINISTER_SUBSCRIPTIONS_TITLE, configuration.ADMIN_ONBOARDING_UUIDS.FLOW_SUBSCRIPTIONS_UUID?.toString()],
+      [ORGANIZE_LEARNERS_TITLE, configuration.ADMIN_ONBOARDING_UUIDS.FLOW_ORGANIZE_LEARNERS_UUID?.toString()],
+      [CUSTOMIZE_REPORTS_TITLE, configuration.ADMIN_ONBOARDING_UUIDS.FLOW_CUSTOMIZE_REPORTS_UUID?.toString()],
+      [SET_UP_PREFERENCES_TITLE, configuration.ADMIN_ONBOARDING_UUIDS.FLOW_PREFERENCES_UUID?.toString()],
+    ]);
+
+    // filter out steps that are turned off for the user
+    const steps = QUICK_START_GUIDE_STEPS.filter(step => {
+      switch (step.title) {
+        case ADMINISTER_SUBSCRIPTIONS_TITLE:
+          return enableSubscriptionManagementScreen;
+        case CUSTOMIZE_REPORTS_TITLE:
+          return enableReportingConfigScreen;
+        default:
+          return true;
+      }
+    });
+
+    if (completedTourFlows) {
+      steps.forEach((step) => {
+        const flowUuid = FLOW_UUID_MAPPING.get(step.title);
+        // @ts-ignore
```

Comment:
Fixing the `getFetchCompletedOnboardingFlows` return type mentioned above should let you remove this `ts-ignore`